dev-environment.yml

```diff
@@ -32,3 +32,5 @@ dependencies:
   - conda-forge::grblas
   - conda-forge::python-louvain
   - conda-forge::websockets
+  - katanagraph/label/dev::katana-cpp
+  - katanagraph/label/dev::katana-python
```

Comment:
We should probably actually release something before becoming a dependency of an external project. ;-) @wkyu2kg maybe talk to the release people to see if we should tag a release in the open package repo.

Reply:
@roshandathathri and @tylershunt Arthur suggested to have a release of katana-cpp and katana-python in the conda repo for them to be the dependency of metagraph plugin. How do you think?

Reply:
No objections but please start a discussion on slack. This may need inputs from many people including Rayees and approval from Chris/Barry.